src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -69,9 +72,9 @@ export class AttachmentUnitFormComponent implements OnChanges {
         competencyLinks: [undefined as CompetencyLectureUnitLink[] | undefined],
     });
     private readonly statusChanges = toSignal(this.form.statusChanges ?? 'INVALID');
-
+    private readonly name = toSignal(this.nameControl?.valueChanges ?? of(''));
     isFormValid = computed(() => {
-        return (this.statusChanges() === 'VALID' || this.fileName()) && !this.isFileTooBig() && this.datePickerComponent()?.isValid();
+        return this.statusChanges() === 'VALID' && !this.isFileTooBig() && this.name() && this.fileName();
```

Comment:
What is the reason for changing the semantics here?

Reply:
`(this.statusChanges() === 'VALID' || this.fileName())` --> `this.statusChanges() === 'VALID' && this.fileName()`:
* statusChanges() evaluates to 'VALID' when I type in a valid name even without an uploaded file
* the rest of the statement would also be true and thus the button activated
* (instead of checking here whether the fileName has been set, I could also include it in the `form: FormGroup` and include it implicitly in statusChanges())

`this.name()` was added:
* I wanted to make sure that `undefined` or the empty string would definetly stop `formValid` from being true
* This is probably not needed bc of the statusChanges, so I could also remove that

`this.datePickerComponent()?.isValid()`:
* I didn't intend to delete that, happend when rebasing on your PR, thanks will add it back